README.md

```diff
@@ -1,8 +1,6 @@
-# Northcoders News API
-
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-
+# READ ME
 
+To run this repo locally you'll need to create two enviroment variables to connect to the two databases locally.
 
```

Comment:
You could be more explicit here - think what you would like to read to get the repo working if you cloned it down.